libs/ngu/carousel/src/test.ts

```diff
@@ -6,8 +6,8 @@ import {
   platformBrowserDynamicTesting
 } from '@angular/platform-browser-dynamic/testing';
 import 'core-js/stable/reflect';
-import 'zone.js/dist/zone';
-import 'zone.js/dist/zone-testing';
+import 'zone.js';
+import 'zone.js/testing';
```

Comment:
These should be provided by the polyfill entry. It's likely you don't need them here.